### ISSUE TITLE
[Feature]Allow ignoring install order alerts

### ISSUE THREAD

**Issue by eXistenZe** [ALIENQuake/ProjectInfinity#40]
**Is your feature request related to a problem? Please describe.**
If a mod has an order that has incompatibilities, it blocks the install.

**Describe the solution you'd like**
If a mod component as a configured incompatible order with another mod, it should give the warning, but never block the install (with a greyed out button). It should always be a user option.

**Comment by ALIENQuake** [comment]
Those rules are often a direct representation of the actual weidu REQUIRE_COMPONENT/REQUIRE_MOD/REQUIRE_PREDICATE statements. Those statements cannot be ignored at all, and are established in order to not break the game. If the user can ignore install order rules set by the modder themself, it would undermine the whole point of this feature.

**Comment by eXistenZe** [comment]
I don't think it was the case. It was a "before x" or "after x". But X was actually a mod that had a lot of components and the one that was out of order (in a duplicate instance) did not matter for anything.

**Comment by ALIENQuake** [comment]
When the mod had split components, the install order rules should cover also those and someday they will.
